### PR TITLE
MES_4469 ADI Part 2 debrief page

### PR DESCRIPTION
**Pull request by edwardwilson**
## Description
Adds placeholder pages for up coming ADI Part 2 schema

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+import { InsomniaMock } from '../../../../shared/mocks/insomnia.mock';
+import { ScreenOrientationMock } from '../../../../shared/mocks/screen-orientation.mock';
+import { TranslateModule, TranslateService } from 'ng2-translate';
+import { fullCompetencyLabels } from '../../../../shared/constants/competencies/competencies';
+import { TestSlotAttributes } from '@dvsa/mes-test-schema/categories/common';
+import { PopulateTestSlotAttributes }
+  from '../../../../modules/tests/journal-data/common/test-slot-attributes/test-slot-attributes.actions';
+import { EndDebrief } from '../../debrief.actions';
+import * as welshTranslations from '../../../../assets/i18n/cy.json';
+import { CAT_BE } from '../../../page-names.constants';
```
Comment (appleabs) [inline src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts]: missing category change

File: src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        StoreModule.forRoot({
+          tests: () => ({
+            currentTest: {
+              slotId: '123',
+            },
+            testStatus: {},
+            startedTests: {
+              123: {
+                testSlotAttributes,
+                category: TestCategory.BE,
```
Comment (appleabs) [inline src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts:97]: missing to-do

File: src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+  describe('endDebrief', () => {
+    it('should dispatch the PersistTests action', () => {
+      component.endDebrief();
+      expect(store$.dispatch).toHaveBeenCalledWith(new EndDebrief);
+    });
+    it('should navigate to PassFinalisationPage when outcome = pass', () => {
+      component.outcome = TestOutcome.PASS;
+      component.endDebrief();
+      expect(navController.push).toHaveBeenCalledWith(CAT_BE.PASS_FINALISATION_PAGE);
```
Comment (appleabs) [inline src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts]: missing category

File: src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+    });
+    it('should navigate to BackToOfficePage when outcome = fail', () => {
+      component.outcome = TestOutcome.FAIL;
+      component.endDebrief();
+      expect(navController.push).toHaveBeenCalledWith(CAT_BE.POST_DEBRIEF_HOLDING_PAGE);
```
Comment (appleabs) [inline src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts]: missing category

File: src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+    });
+    it('should navigate to the BackToOfficePage when outcomes = terminated', () => {
+      component.outcome = 'Terminated';
+      component.endDebrief();
+      expect(navController.push).toHaveBeenCalledWith(CAT_BE.POST_DEBRIEF_HOLDING_PAGE);
```
Comment (appleabs) [inline src/pages/debrief/cat-adi-part2/__tests__/debrief.cat-adi-part2.page.spec.ts]: missing category

**Review by appleabs — APPROVED** [review]

**Review by mgoldson — APPROVED** [review]

**Pull request merged**